SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -600,17 +600,12 @@ void ASpatialDebugger::DrawDebug(UCanvas* Canvas, APlayerController* /* Controll
 		{
 			const TWeakObjectPtr<AActor> Actor = EntityActorPair.Value;
 			const Worker_EntityId EntityId = EntityActorPair.Key;
-
-			if (Actor != nullptr)
+			FVector2D ScreenLocation;
+			if (Actor != nullptr && !ProjectActorToScreen(Actor->GetActorLocation(), PlayerLocation, ScreenLocation, Canvas))
```

Comment:
This is not right, if actor == nullptr DrawTag will dereference a nullptr here `Actor->GetName()`

Reply:
I would suggest writing the following, reduces the amount of control flow statements too, so a bit easier to read.
```
FVector2D ScreenLocation;
if (Actor != nullptr && ProjectActorToScreen(Actor->GetActorLocation(), PlayerLocation, ScreenLocation, Canvas))
{
	DrawTag(Canvas, ScreenLocation, EntityId, Actor->GetName(), true /*bCentre*/);
}
```